library/build.gradle

```diff
@@ -33,7 +33,7 @@ processResources {
 
 // Run this task to bundle all needed dependency
 task bundle(type: Jar) {
-    baseName = archivesBaseName + '-all'
+    baseName = archivesBaseName + '-' + version + '-all'
```

Comment:
It already does this by default when building the jar. This will result in NAME-VERSION-VERSION-all